.gitlab-ci.yml

```diff
@@ -65,7 +65,7 @@ trigger-agent-build:
     - export AGENT_REQUIREMENTS_CHANGED="false"
     - git fetch origin master
     - BASE_REF=$(git merge-base HEAD FETCH_HEAD)
-    - git --no-pager diff --exit-code --name-only HEAD $BASE_REF -- datadog_checks_base/datadog_checks/base/data/agent_requirements.in || export AGENT_REQUIREMENTS_CHANGED="true" || true
+    - git --no-pager diff --exit-code --name-only HEAD $BASE_REF -- .deps || export AGENT_REQUIREMENTS_CHANGED="true" || true
     - export RELEASE_BRANCH=""
     - |
       if [[ "$AGENT_REQUIREMENTS_CHANGED" = "true" ]]; then
```

Comment:
nit - there's a reference to `agent_requirements.in` line 73

Reply:
Thanks! Just changed that as well as renaming the `AGENT_REQUIREMENTS_CHANGED` to avoid misleading references to the agent_requirements file.